mage_ai/orchestration/db/__init__.py

```diff
@@ -39,7 +39,8 @@
             db_connection_url = f'sqlite:///{TEST_DB}'
         elif os.path.exists('mage_ai/orchestration/db/'):
             # For local dev environment
-            db_connection_url = 'sqlite:///mage_ai/orchestration/db/mage-ai.db'
+            path = os.path.abspath(os.path.join('mage_ai/orchestration/db', 'mage-ai.db'))
```

Comment:
What's this change for?

Reply:
it's something to fix sqlite on dev, but i'll move it to a separate PR